uint32 time

float32 roll
float32 pitch
float32 heading

float32 rollAcc
float32 pitchAcc
float32 headingAcc